util/unifyfs/src/unifyfs-rm.c

```diff
@@ -808,10 +810,50 @@ int unifyfs_start_servers(unifyfs_resource_t* resource,
     }
 
     if (args->script != NULL) {
-        return script_launch(resource, args);
+        return_val=script_launch(resource, args);
     } else {
-        return resource_managers[resource->rm].launch(resource, args);
+        return_val=resource_managers[resource->rm].launch(resource, args);
     }
+
+    // here we will check the status file that all the servers are
+    // up and talking to each other and are happy.
+
+    // check for a --stage-out= argument.  If there is one, throw
+    // a warning that a --stage-out argument with a server launch
+    // doesn't make any sense and is not supported at this time.
+    if(args->stage_out != NULL){ 
```

Comment:
All CLI arg checking should be done in `unifyfs.c:parse_cmd_arguments()`. That includes checking for existence of the provided manifest file.